### PR TITLE
feat : added typography, list and toast

### PR DESCRIPTION
**Pull request by ArjunJunna**
added the typography, list and toast components.

### CODE REVIEW

**Review by sanathkumar26700 — APPROVED** [review]
✅ Clean code
✅ Good UI
✅ Everything is working fine

File: components/toast/toast.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        <h1 class="main-heading">Toast</h1>
+        <h4 class="heading-info">
+          Toasts are usually used to put out a notification for the user. It can be even called as a notify bar and snack bar.
+        </h4>
+        <h3 class="sub-heading">
+          Below is an example for toast.
+        </h3>
+        <section class="component">
+          <div class="toast">
+              <button class='btn-toast'>A toast message  <i class="fas fa-times icon-dismiss"></i></button>
```
Comment (mikesch803) [inline components/toast/toast.html:37]: your whole toast is wrapped in the btn, which should be in div. 
for reference you can check https://getbootstrap.com/docs/4.3/components/toasts/

Comment (ArjunJunna) [reply components/toast/toast.html:37]: sure will make the changes. thank you

File: css/layout/main.css [hunk 2]
```diff
@@ -13,7 +13,20 @@ main h3 {
 }
 main h5{
   margin: 1rem 0 0 4rem;
```
Comment (fahd23) [inline css/layout/main.css]: Remove this commented code if possiblr

**Review by fahd23 — APPROVED** [review]
Looks good to merge

File: css/layout/main.css (hunk 2 above)
Comment (ArjunJunna) [reply css/layout/main.css]: sure will remove that. thanks.

**Pull request merged**